about.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+        <script src="https://kit.fontawesome.com/187f763109.js" crossorigin="anonymous"></script>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">          
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+    <link rel="stylesheet" href="https://use.typekit.net/ntt5mmb.css">
+    <title>Poem catcher - A soulful Journey of a Lost Poet  </title>
+    <link rel="shortcut icon" type="img/png" href="./src/poetry-title-img.png">
+</head>
+<body>
+         <nav class=" nav-bar">
+    <nav class="top-nav">
+        <ul>
+            <li><a href="#" target="_blank"><i class="fa-brands fa-facebook-f"></i></a></li>
+            <li><a href="#" target="_blank"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My page</a></li>
+            <li><a href="#">Log Out</a></li>
+        </ul>
+    </nav>
+    <nav class="bottom-nav">
+        <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="logo-wrap">
+             <a href="./index.html"><img class="logo" src="./src/poetry-title-img.png" alt="Poetry logo"></a>
+
+            </div>
+            <ul class="nav-menu">
+                <li class="nav-item"><a class = "nav-link portfolio-link" href="#">About</a></li>
+                <li class="nav-item"><a class =  " nav-link about-link" href="#">Poems</a></li>
+                <li class="nav-item"><a class = "nav-link contact-link" href="#">Stories</a></li>
+                <li class="nav-item"><a class = "nav-link contact-link" href="#">Publications</a></li>
+                <li class="nav-item"><a class = "nav-link contact-link" href="#">Donate</a></li>
+    
+            </ul>
+    </nav>
+            </nav>
+ <section>
+    <div class="landingpage-wrap">
+    <div>
+        <div>
+            <div class="about-welcome-text">
+                <P>"Hello! Poetic Souls"</P>
+                <h1>
+                   Slam Poetry Jam
+                    2022
+                </h1>
```

Comment:
- [ ] [OPTIONAL] Kindly reduce the font size on the desktop about page. Ensure you implement the required margin right, left, and center so it looks more like the template provided.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-06-25 16-00-06](https://user-images.githubusercontent.com/58692590/175779445-44fc26d3-6eac-4aa0-b808-19f53a1a6469.png) | ![Screenshot from 2022-06-25 16-02-57](https://user-images.githubusercontent.com/58692590/175779464-b775af8a-854f-40c2-90a6-c2f79a99bb6a.png)